### DISCUSSION TITLE
The number by which the global ranking improved should be shown with minus instead of plus sign

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#27263] [Ideas]
Currently when achieving a good score which decreases your global ranking by e.g. 1000 (which is an improvement, a good thing) it shows up in the top right as +1000 instead of -1000 after the results screen is opened. This is illogical. I think the global ranking improvement should be displayed with a minus sign and pp (if it improved) with a plus sign as it is already. Both numbers should still be displayed in green when global ranking decreased (improvement) and pp increased (improvement).

Perhaps plus sign was used because minus sign is associated with decrease which might be associated with a bad result? I don't think that's a good reason because the color green already indicates that this score was in fact good. No need to encode that meaning in +/- signs.

Likewise if the score was bad and global ranking increased and pp decreased, plus sign and minus sign should be used, respectively, with the text in red color.

It also simply looks confusing if a plus sign is shown but the number actually decreases.

**Comment by TwoQuantumBits** [comment]
I think it's due to the fact that stable had + for rank up, so the same was applied to lazer. Will wait for other people's feedback on this, though.

**Comment by Joehuu** [comment]
I agree with you that using plus is wrong but also is using minus in isolation (often means something negative and coloring the text green just makes it confusing). There should be an arrow accompanied like what results screen does:

<img width="1029" alt="Screenshot 2024-02-19 at 10 21 48 PM" src="https://github.com/ppy/osu/assets/35318437/11e425da-13c3-4f61-8219-c8ceb69db8dc">

Ideally, ranking changes should only have arrows as it's a special number saying lower is better. This [WIP web PR that shows ranking changes on performance ranking](https://github.com/ppy/osu-web/pull/10597) is what I expect:
<img width="175" alt="Screenshot 2024-02-19 at 10 43 58 PM" src="https://github.com/ppy/osu/assets/35318437/6f212115-4fb5-4b34-b84a-5532678090ae">

**Comment by bdach** [comment]
I don't get any of this.

Gaining ranks should be shown with a plus sign and green colour because _that matches everything else elsewhere_. We've been over this so many times now. Yes it "mathematically" doesn't make sense but _intuitively_ it makes sense.

I don't see what adding arrows or inverting the directionality of the sign does other than introduce further confusion. All I want is consistent UX: gains are green and with a plus sign. Losses are red and with a minus sign.

I'd rather just remove the sign and keep the colour if I had to change _anything_ there. But I don't think that's necessary.

**Reply by Joehuu** [reply]
If I wasn't clear in my post, here is what I prefer:
1. replace sign with arrow / remove sign and add arrow to the right of main value
2. add arrow to the right of main value (to have consistent UX with results screen, which is want you want)
3. no change
4. invert sign and add arrow
5. remove sign (your proposal)
6. invert sign (op's proposal)

4 is above 5 and 6 simply because if color is not present or one is colorblind, there is no second indicator of the result being positive/negative. I still don't like the invert idea so changing nothing beats it.
